web-marketplace/package.json

```diff
@@ -10,8 +10,8 @@
     "@cosmjs/cosmwasm-stargate": "^0.31.0",
     "@cosmjs/launchpad": "^0.27.0",
     "@cosmjs/stargate": "^0.27.0",
-    "@cosmos-kit/keplr-mobile": "^2.6.7",
-    "@cosmos-kit/react-lite": "^2.6.7",
+    "@cosmos-kit/keplr-mobile": "^2.9.0",
+    "@cosmos-kit/react-lite": "^2.9.0",
```

Comment:
this wasn't needed to fix the issue but while debugging, I upgraded those packages that we use to connect via wallet connect